### PR TITLE
Adapt arrow speed to line loading factor

### PR DESCRIPTION
**Pull request by geofjamg**
Signed-off-by: Geoffroy Jamgotchian <geoffroy.jamgotchian@rte-france.com>

### CODE REVIEW

File: src/components/network/line-layer.js [hunk 1]
```diff
@@ -50,44 +49,124 @@ function getArrowDirection(p) {
...
+            return ArrowSpeed.CRAZY;
+        }
+    }
+}
+
+function getArrowSpeed(line, lineFlowAlertThreshold) {
+    const speed1 = getArrowSpeedOfSide(
+        line.permanentLimit1,
+        line.i1,
+        lineFlowAlertThreshold
```
Comment (FranckLecuyer) [inline src/components/network/line-layer.js:146]: third parameter lineFlowAlertThreshold not used in getArrowSpeedOfSide function
you can also remove it as second parameter of getArrowSpeed function

File: src/components/network/line-layer.js [hunk 2]
```diff
@@ -418,9 +498,19 @@ class LineLayer extends CompositeLayer {
...
+                            nominalVoltageColor,
+                            this.props
+                        ),
                     getSize: 700,
-                    getSpeedFactor: 3,
+                    getSpeedFactor: (arrow) =>
+                        getArrowSpeedFactor(
+                            getArrowSpeed(
+                                arrow.line,
+                                this.props.lineFlowAlertThreshold
```
Comment (FranckLecuyer) [inline src/components/network/line-layer.js:511]: parameter 'this.props.lineFlowAlertThreshold' not used

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/components/network/line-layer.js (hunk 1 above)
Comment (geofjamg) [reply src/components/network/line-layer.js]: Fixed

File: src/components/network/line-layer.js (hunk 2 above)
Comment (geofjamg) [reply src/components/network/line-layer.js]: Fixed

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**